src/main/java/mezz/jei/network/PacketHandler.java

```diff
@@ -45,7 +45,7 @@ public void onPacket(NetworkEvent.ClientCustomPayloadEvent event) {
 			PacketIdServer packetId = PacketIdServer.VALUES[packetIdOrdinal];
 			IPacketJeiHandler packetHandler = serverHandlers.get(packetId);
 			packetHandler.readPacketData(packetBuffer, player);
-		} catch (RuntimeException e) {
+		} catch (Throwable e) {
```

Comment:
I think catch `RuntimeException` is actually good here, we don't want to catch memory errors and other serious exceptions that could make debugging a crash very confusing.